numpy/core/tests/test_function_base.py

```diff
@@ -301,9 +301,9 @@ def __mul__(self, other):
     def test_denormal_numbers(self):
         # Regression test for gh-5437. Will probably fail when compiled
         # with ICC, which flushes denormals to zero
-        for dt in (dtype(f) for f in typecodes['Float']):
-            stop = finfo(dt).tiny * finfo(dt).resolution
-            assert_(any(linspace(0, stop, 10, endpoint=False, dtype=dt)))
+        for ftype in sctypes['float']:
+            stop = nextafter(ftype(0), ftype(1)) * 5  # A denormal number
```

Comment:
If this was 0 before, then an assertion (or perhaps better, an exception to make the test error) that it is not zero now would probably be a good thing.